Sources/StreamChat/Utils/Cached.swift

```diff
@@ -38,11 +40,14 @@ class Cached<T> {
         set { computeValue = newValue }
     }
 
-    @Atomic private var _cached: T?
+    private let queue = DispatchQueue(label: "io.getstream.cached")
+    private var _cached: T?
```

Comment:
Testing if CI test instability has something to do with exclusive access requirement from the Swift runtime (`@Atomic` can trigger it)